src/components/login-package/UserInput.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+import {ACCESS_TOKEN} from "../../constants/constants";
+import {Redirect} from "react-router";
+
+export default class UserInput extends React.Component {
+    state = {
```

Comment:
would be better to set "this.state" in constructor

Reply:
Done